gammapy/makers/tests/test_utils.py

```diff
@@ -155,7 +155,9 @@ def bkg_3d_custom(symmetry="constant"):
     energy_axis = MapAxis.from_energy_edges([0.1, 10, 1000] * u.TeV)
     fov_lon_axis = MapAxis.from_edges([-3, -1, 1, 3] * u.deg, name="fov_lon")
     fov_lat_axis = MapAxis.from_edges([-3, -1, 1, 3] * u.deg, name="fov_lat")
-    return Background3D(
+    Background3D_extra = Background3D
+    Background3D_extra.fill_value = None #allow extrapolation for symetry tests
```

Comment:
One last idea: I noticed that we only define `default_interp_kwargs`, but no instance level `interp_kwargs`. So maybe we just add:

```python3

class IRF:
    def __init__(self, ..., interp_kwargs=None):
        if interp_kwargs is None:
            interp_kwargs = self.interp_kwargs_default.copy()
        self.interp_kwargs = interp_kwargs
```